pointcloud/basic_octree_test.go

```diff
@@ -697,3 +699,99 @@ func TestCachedMaxProbability(t *testing.T) {
 		test.That(t, mp, test.ShouldEqual, -2)
 	})
 }
+
+func TestBasicOctreeTransform(t *testing.T) {
```

Comment:
[q] Are you expecting to add tests for AlmostEqual and ToPoints as well?

Reply:
This test fully covers ToPoints as well.

Reply:
I can add tests for AlmostEqual